website/docs/advanced/tenancy.md

```diff
@@ -11,37 +11,65 @@ This feature is in alpha. Use at your own risk.
+
+### Important considerations
+
+-   Upon creating another tenant, a new schema will be created by cloning the `template` schema. This special schema is like a tenant with no data created in it. Cloning an existing schema instead of creating a new one and running migrations on it is done for efficiency purposes.
 
-authentik manages tenants by storing data for each tenant in a separate PostgreSQL schema.
+-   Data stored in Redis (cache, tasks, locks) will usually get its keys prefixed by the `schema_name`.
 
-This feature needs to be enabled with the `AUTHENTIK_TENANTS__ENABLED=true`. You also need to set `AUTHENTIK_TENANTS__API_KEY` to a random string, which will be used to authenticate to the tenancy API. This key will allow the creation of recovery keys for every tenant hosted by authentik, store it securely. You will also need to disable the embedded outpost with `AUTHENTIK_OUTPOSTS__DISABLE_EMBEDDED_OUTPOST=true` as it is not supported with tenants.
+-   Files are stored by-tenant, under a `schema_name` directory. For example, `/media/t_example`. The same is true regardless of the storage backend.
 
 ## Usage
 
-Tenants can be created using the API routes associated. Search for `tenant` in the [API browser](../../developer-docs/api/) for the available endpoints.
+To create one or more additional tenants (beyond the default tenant) use the following instructions.
 
-When creating a tenant, you must specify a `name`, used for display purposes, and a `schema_name`, used to create the PostgreSQL schema associated with the tenant. That `schema_name` must start with `t_` and not be more than 64 characters long.
+### 1. Configure authentik to allow multiple tenants
 
-There is always at least one tenant, `public`. This is the default tenant and cannot be deleted. Despite its name, it is not freely available to the world. Instead, it is stored in the `public` schema of the PostgreSQL database.
+First, enable with the `AUTHENTIK_TENANTS__ENABLED=true` [configuration setting](../installation/configuration.mdx).
 
-By default, all requests that do not explicitly belong to a tenant are redirected to the default tenant. Thus, after creating a tenant, you must associate a domain for which incoming requests will be redirected to said tenant. You can do so with API endpoints. After creating a domain `example.org` that is associated to the tenant `t_example`, all requests made to `example.org` will use the `t_example` tenant. However, requests made to `authentik.tld`, `subdomain.example.org` and all other domains will use the default tenant.
+Then set `AUTHENTIK_TENANTS__API_KEY` to a random string, which will be used to authenticate to the tenancy API. This key allows the creation of recovery keys for every tenant hosted by authentik, so be sure to _store it securely_. Be aware that creating a recovery key allows access to all data stored inside a tenant.
```

Comment:
I read "first/then" awkwardly.  Maybe eliminate "First, " in the first sentence?  Or numbers?

> 1. Enable multi-tenancy with the ...
> 2. Set the `AUTHENTIK_TENANTS__API_KEY` to a random string.  This string will be used to ...

In the second task, I separated the _task_ from the _reason_ to make it clear that one is an action, the other an explanation.  The context is supported by their being in the same paragraph.

Reply:
Yes! I want more of this in our docs... helpful ways to do things... so that we don't come across assuming everyone is a senior dev. ;-) (Though it is challenging to find the right balance, and not be too pedantic!)

Reply:
My usual go-to is `tr -dc 'a-zA-Z0-9' < /dev/urandom | head -c 32; echo`, but we can provide the more generic `pwgen 64 1`